exercises/sieve/test/test_sieve.c

```diff
@@ -15,22 +16,18 @@ void tearDown(void)
 void test_no_primes_under_two(void)
 {
    const unsigned int limit = 1;
-   const primes_array_t expected_prime_array = { 0 };
    const unsigned int expected_prime_count = 0;
    unsigned int result_prime_count;
 
    memset(result_array, 0, sizeof(result_array));
 
    result_prime_count = sieve(limit, result_array);
    TEST_ASSERT_EQUAL(expected_prime_count, result_prime_count);
-   TEST_ASSERT_EQUAL(0,
-                     memcmp(expected_prime_array, result_array,
-                            sizeof(result_array)));
 }
 
 void test_find_first_prime(void)
 {
-   TEST_IGNORE();               // delete this line to run test
+   // delete this line to run test
```

Comment:
Is this correct?

Reply:
Okay, it looks like these were removed for testing. Be sure to add them back in.